app/src/main/res/layout/fragment_profile.xml

```diff
@@ -1,6 +1,206 @@
+                    android:fontFamily="@font/wellfleet"
+                    android:inputType="phone"
+                    android:maxLength="10"
+                    android:textSize="20sp" />
+            </com.google.android.material.textfield.TextInputLayout>
+
+            <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/textViewRadioGroup"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="15dp"
+                android:fontFamily="@font/wellfleet"
+                android:text="@string/tv_employee_status"
+                android:textAlignment="center"
+                android:textColor="@color/redSimformColor"
+                android:textSize="19sp"
+                app:layout_constraintStart_toStartOf="@id/editTextNameMaterialLayout"
+                app:layout_constraintEnd_toEndOf="@id/editTextNameMaterialLayout"
+                app:layout_constraintTop_toBottomOf="@id/editTextDepartmentMaterialLayout"/>
+
+            <RadioGroup
+                android:id="@+id/radioGroup"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                android:layout_marginTop="5dp"
+                app:layout_constraintStart_toStartOf="@id/editTextNameMaterialLayout"
+                app:layout_constraintEnd_toEndOf="@id/editTextNameMaterialLayout"
+                app:layout_constraintTop_toBottomOf="@id/textViewRadioGroup">
+
+                <RadioButton
+                    android:id="@+id/radioButtonTrainee"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="5dp"
+                    android:fontFamily="@font/wellfleet"
+                    android:text="@string/radiobtn_trainee"
+                    android:textColor="@color/redSimformColor"
+                    android:textSize="20sp" />
+
+                <RadioButton
+                    android:id="@+id/radioButtonEmployee"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_margin="5dp"
+                    android:fontFamily="@font/wellfleet"
+                    android:text="@string/radiobtn_employee"
+                    android:textColor="@color/redSimformColor"
+                    android:textSize="20sp" />
+            </RadioGroup>
+
+            <androidx.appcompat.widget.AppCompatButton
+                android:id="@+id/btnEditProfile"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="5dp"
+                android:background="@drawable/button_corner"
+                android:fontFamily="@font/wellfleet"
```

Comment:
Can you please check this fontFamily is working or not?

Reply:
Yes Its Working

Reply:
I have only Downloaded